docs/contributing.md

```diff
@@ -105,16 +108,18 @@ number**.
 Create a local branch that corresponds to the issue. To easily
 identify the purpose of branches different keywords must be used:
 
-* Patch branches must be named **patch-[issue number]-[short description]**
-* Feature branches must be named **feature-[issue number]-[short description]**
+* Contributions targeted to the **current** release must be named 
+  **current-[issue number]-[short description]**
+* Contributions targeted to the **next** major release must be named
+  **next-[issue number]-[short description]**
 
 If you create these branches by hand please check the spelling because otherwise
 the test automation might misidentify your branch. There are utility scripts to
 fetch master and create these branches for you:
 
 ```bash
-./tools/create-patch-branch <unique issue number> '<short_change_message>'
-./tools/create-feature-branch <unique issue number> '<short_change_message>'
+./tools/create-current-branch <unique issue number> '<short_change_message>'
+./tools/create-next-branch <unique issue number> '<short_change_message>'
```

Comment:
these scripts are currently still named `create-patch-branch` and `create-feature-branch`